compiler/Eta/BasicTypes/Literal.hs

```diff
@@ -137,8 +137,8 @@ They only get converted into real Core,
 during the CorePrep phase, although TidyPgm looks ahead at what the
 core will be, so that it can see whether it involves CAFs.
 
-When we initally build an Integer literal, notably when
-deserialising it from an interface file (see the Binary instance
+When we initially build an Integer literal, notably when
+deserializing it from an interface file (see the Binary instance
```

Comment:
Google suggested this. Generally I leave `-ise` British variants alone. But the Internet agrees that this `-z`/`-s` variation doesn't exist.